docs/resource-policy/policy/balloons.md

```diff
@@ -154,6 +154,9 @@ Balloons policy parameters:
     separate `cpu.classes` objects, see below.
   - `preferCloseToDevices`: prefer creating new balloons close to
     listed devices. List of strings
+  - `coreType`:  specifies preferences of the core type which could
+    be power either efficient (`efficient`) or high performance
```

Comment:
typo/thinko: 'could be either power efficient ('`efficient`) or high performance'